zkvm/precompiles/src/secp256k1.rs

```diff
@@ -190,22 +190,11 @@ fn double_and_add_base(
 /// use `ecrecover`.
 pub fn unconstrained_ecrecover(sig: &[u8; 65], msg_hash: &[u8; 32]) -> ([u8; 33], Scalar) {
     unconstrained! {
```

Comment:
I put it there because I reasoned that the computations are not directly used by the output, so there is no reason to have the VM constrain these values. @ctian1 said it was okay for now, although there is some overhead with `unconstrained!`. We could benchmark it to determine whether or not we should remove it.

Reply:
I just checked: the cycle count slightly increases without the `unconstrained!` block, from 69957 to 70110.